### ISSUE TITLE
Query 32-bit application data

### ISSUE THREAD

**Issue by pd4d10** [desktop/registry-js#172]
Is there any approach to query 32-bit application data?

Reference: https://docs.microsoft.com/en-us/windows/win32/sysinfo/32-bit-and-64-bit-application-data-in-the-registry

**Comment by CyberSecDemon** [comment]
was this ever resolved?

**Comment by sergiou87** [comment]
Nope, as far as I can see the implementation is still the same in that regard.

**Comment by CyberSecDemon** [comment]
Yeah I checked after posting I've reworked this for 32 as well in our codebase. Also another issue I've also added RegNotifyChabgeKeyValue. May request PR
